pkg/api/schema/schema.proto

```diff
@@ -1439,6 +1454,19 @@ message RetryInfo {
   int32 retry_delay = 1;
 }
 
+message TruncateDatabaseRequest {
+  // Database name
+  string database = 1;
+
+  // Retention Period of data
+  NullableMilliseconds retentionPeriod = 2;
+}
+
+message TruncateDatabaseResponse {
+  // Database name
+  string database = 1;
```

Comment:
ideally we should return the older txID still present in the database after truncation

Reply:
I can try adding that later, will keep this as lower priority